cmd/m4d/main.go

```diff
@@ -0,0 +1,43 @@
+package main // import "github.com/ibm/the-mesh-for-data/cmd/m4d"
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func rootCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "m4d",
+		Short:         "Mesh for Data CLI",
+		SilenceErrors: true,
+		SilenceUsage:  true,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			_ = viper.BindPFlags(cmd.Flags())
+		},
+	}
+
+	// Add subcommands
+	cmd.AddCommand(
+		newTaxonomyCmd(),
+	)
+
+	cobra.OnInitialize(initConfig)
+	viper.SetEnvKeyReplacer(strings.NewReplacer("-", "_"))
+	return cmd
+}
+
+func initConfig() {
+	viper.AutomaticEnv()
+}
+
+func main() {
```

Comment:
Is this a tool that is meant to be run from the command line?  Who is meant to use it? Where are the user instructions?